### ISSUE TITLE
Story list violates color contrast

### ISSUE THREAD

**Issue by nmain** [storybookjs/storybook#4688]
**Describe the bug**
The list of components on the left side of storybook has poor contrast.

**To Reproduce**
Load any storybook, and expand the left side.

**Expected behavior**
Color contrast should pass WCAG.

**Screenshots**
In this screenshot, look at TunnelVision, TunnelVisionExtended, and TunnelVision.shouldAllowClose
![image](https://user-images.githubusercontent.com/24611942/47861009-33611c00-ddc8-11e8-8125-1a6acd7fd796.png)

**System:**
 - OS: Windows 10
 - Browser: Chrome
 - Version: Whatever is on https://building.coursera.org/ right now

**Additional context**
The current styles are as follows:
```
background: rgb(247, 247, 247)
color: rgb(130, 130, 130)
font-size: 13px
font-family: San Francisco
```
This gives a very weak score of 3.57
![image](https://user-images.githubusercontent.com/24611942/47861215-ad91a080-ddc8-11e8-9b71-f3becbe200c6.png)

**Comment by Muhammad-Ibrahim** [comment]
I want to start working on this issue

**Comment by igor-dv** [comment]
@Muhammad-Ibrahim that's great 👍

**Comment by Muhammad-Ibrahim** [comment]
@igor-dv  is that ok?
https://contrastchecker.com/?swatch_session=1541243991104

![image](https://user-images.githubusercontent.com/8923156/47952296-a22fa800-df75-11e8-9e2d-7d80ba7d0f76.png)

**Comment by igor-dv** [comment]
Contrastchecke says it's ok. Who am I to disagree? =)

**Comment by Muhammad-Ibrahim** [comment]
Ok, cool 😆 

Now, I can't find the class `.css-1b2u8sb` styling

I already did that:
- forked the repo.
- run `yarn install` then `yarn bootstrap --core`
- searched inside the folder about the class name and found 1 result at \storybook\lib\components\src\navigation\__snapshots__\menu_link.stories.storyshot

**Comment by igor-dv** [comment]
That's an `emotion` css framework, and classes are generated dynamically.  You just need to find the MenuLink component and fix it there

**Comment by reddysridhar53** [comment]
Is this still open?

**Comment by igor-dv** [comment]
I think it is. 
@Muhammad-Ibrahim ?

**Comment by Muhammad-Ibrahim** [comment]
yes it is, @reddysridhar53  @igor-dv

**Comment by wyseguyonline** [comment]
Is this issue still open?

**Comment by igor-dv** [comment]
Yep

**Comment by untempered** [comment]
Looks like this issue can be closed since the current version (5.1.0) has much different text color in the sidebar and now passes WCAG. 
![image](https://user-images.githubusercontent.com/19578001/60043064-d2eb5b80-9684-11e9-9b6c-ee198ea780e0.png) Can someone confirm and close?
